src/main/resources/templates/java/test/gradle/projectTemplate/readme.md

```diff
@@ -0,0 +1,19 @@
+## Test Repository instructions
+
+#### Test tool & command
+Tests will be run using the command ./gradlew clean test.
```

Comment:
Yeah. This won't work unless you include the Gradle wrapper.
If you do not include it, it has to be created first. (Which you should explain here)

Reply:
Changed to `gradle clean test`.